docs/reference/ilm/actions/ilm-allocate.asciidoc

```diff
@@ -0,0 +1,118 @@
+[[ilm-allocate]]
+=== Allocate
+
+Phases allowed: warm, cold.
+
+The Allocate action allows you to specify which nodes are allowed to host the
```

Comment:
I wouldn't capitalize here. If you think the text needs to be set off, I'd use `allocate`.
```suggestion
The allocate action allows you to specify which nodes are allowed to host the
```